<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-OH8IUHA" failures="4" tests="7" name="HRS" time="1061.729" errors="0" timestamp="2023-07-05T19:39:45 IST">
  <testcase classname="com.hrs.test.ClinicianConnect" name="verifyPatientListGeneralDetailsDisplay" time="69.591"/>
  <testcase classname="com.hrs.test.ClinicianConnect" name="verifyMetricsTabGeneralDisplay" time="231.319">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Care Plan Metrics were successfully saved.] but found [An error occurred while saving the Care Plan Metrics. Please try again later.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Care Plan Metrics were successfully saved.] but found [An error occurred while saving the Care Plan Metrics. Please try again later.]
at com.hrs.test.ClinicianConnect.verifyMetricsTabGeneralDisplay(ClinicianConnect.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyMetricsTabGeneralDisplay -->
  <testcase classname="com.hrs.test.ClinicianConnect" name="verifyAddReminderModalGeneralDisplay" time="13.461">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index 1 out of bounds for length 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 1 out of bounds for length 1
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
at java.base/java.util.Objects.checkIndex(Objects.java:372)
at java.base/java.util.ArrayList.get(ArrayList.java:459)
at com.business.hrs.HRSBusinessLogic.validateAddReminderModalGeneralDisplay(HRSBusinessLogic.java:364)
at com.hrs.test.ClinicianConnect.verifyAddReminderModalGeneralDisplay(ClinicianConnect.java:81)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verifyAddReminderModalGeneralDisplay -->
  <testcase classname="com.hrs.test.ClinicianConnect" name="verifyAddMedicationFrequencyOnceDaily" time="206.211">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [LIQUAEMIN SODIUM (Essential)] but found []">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [LIQUAEMIN SODIUM (Essential)] but found []
at com.hrs.test.ClinicianConnect.verifyAddMedicationFrequencyOnceDaily(ClinicianConnect.java:116)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyAddMedicationFrequencyOnceDaily -->
  <testcase classname="com.hrs.test.ClinicianConnect" name="verifyPatientConnectMobileLoginCodesLoggingIntoPCM" time="43.29"/>
  <testcase classname="com.hrs.test.Homepage" name="verifyhomePage" time="442.873">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Education Button  is not displayed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Education Button  is not displayed expected [true] but found [false]
at com.hrs.test.Homepage.verifyhomePage(Homepage.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyhomePage -->
  <testcase classname="com.hrs.test.Homepage" name="verifyBloodPressure" time="0.0">
    <skipped/>
  </testcase> <!-- verifyBloodPressure -->
  <testcase classname="com.hrs.test.BaseTest" name="@AfterTest afterTest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
</testsuite> <!-- HRS -->
